README.md

```diff
@@ -1,2 +1,56 @@
+
+- **Creating a virtual environment**
+
+   In your console, navigate to the folder where you installed the project. Run the folowing command:
+   
+   `python -m venv <path_to_venv>`
+   
+   `path_to_venv` can be either a relative or absolute path. Make sure NOT to use the angled brackets when writing the path. For example, `./venv` will create a new folder for your virtual environment on your current directory called venv. Please note that it may take a few minutes for the virtual envronment to be created, depending on how good the computer running the command is.
+   
+- **Running the virtual environment**
+
+   Now that the venv is installed, you should see a folder called `Scripts` with an activation script inside. You can activate the venv by typing `./<path_to_venv>/Scripts/activate` in your console.
+
+- **Download the required assets from requirements.txt**
+
+  In requirements.txt you'll find a list of required assets and libraries that need to be downoaded first in order for the project to work correctly. You can install everything in that file to your current working environment (in this case the venv) by going to the console and writing `pip install -r requirements.txt`
+
+- **Create migrations**
+
+  Django uses what's called *models* to create a single, definitive source of information about your data. They contain the essential fields and behaviors of the data you’re storing, and generally map to a single database table.
+
+  This project already contains the finished models that it needs, however you first need to create the migrations required to create the database in the first place. For that, go to your console and write `python -m manage.py makemigrations`. This will create the migrations required for a database to be created and for the models to map to said database. By default, the database created is an SQLite3 database.
+
+- **Run the migrations**
+  
+  Finally, to apply the migration changes, go to your console and run the command `python -m manage.py migrate`. This will create a database for your data to be stored.
+
+  **YOUR DATABASE MAY CONTAIN SENSITIVE INFORMATION, SO NEVER SHARE IT WITH ANYONE YOU DON'T TRUST**
+
+- **Enable stripe**
+
+  Payments are processed using Stripe. You will need to make an account on Stripe first by going [here](https://dashboard.stripe.com/register), in order to get your own API keys. Once your account is created, you will need to activate it, and then go to your dashboard. In the middle of the page, click on "API keys for developers".
+  
+  ![εικόνα](https://github.com/ArceusLegend/Ecommerce/assets/109414442/1a767a99-9156-40cc-a921-aa99aab07ca9)
+
+  Now go to the project, and navigate to core/settings.py Scroll all the way to the bottom to find the `PUBLISHABLE_KEY`, `SECRET_KEY`, and `STRIPE_ENDPOINT_SECRET` constants. Replace these with your own keys; the publishable and secret keys can be found in the API keys tab:
```

Comment:
I'd suggest to never put any secret keys in the code (especially tracked files). It's better to export them into the target environment. One suggestion is to use an `.env` file to locally setup your envvars. You can create an `.env.template` without the key and track this, but add the actual `.env` file in `.gitignore`. Also update the settings and readme to reflect this change

Reply:
I moved the stripe keys and the django secret key from settings.py to an untracked .env file. I'll update the readme to reflect these changes